public/alcom/releases/index.hbs

```diff
@@ -0,0 +1,58 @@
+{{#> page
+title="Releases - ALCOM"
+icon="/resources/alcom.svg"
+with="social"
+}}
+
+{{#*inline "social"}}
+<meta property="og:title" content="ALCOM - {{common.download}}" />
+<meta property="og:type" content="website" />
+<meta property="og:url" content="https://vrc-get.anatawa12.com/{{locale.code}}/alcom/releases" />
+<meta property="og:image" content="https://vrc-get.anatawa12.com/resources/alcom.svg" />
+<meta property="og:site_name" content="ALCOM" />
+<meta property="og:description" content="{{home.alcomDescription}}" />
+{{/inline}}
+
+{{#*inline "content"}}
+
+<br />
+
+<div>
+    <a class="p-button has-icon" href="/{{locale.code}}/alcom">
+        <i class="p-icon--chevron-left"></i>
+        <span>ALCOM</span>
+    </a>
+</div>
+
+<h2 class="inline-block u-no-margin--bottom me-1">{{common.releases}}</h2>
+<aside class="p-accordion mt-2">
+    <ul class="p-accordion__list" id="releases-list">
+        {{#each __releases}}
+        {{#if isGui}}
+        <li class="p-accordion__group">
+            <h4 role="heading" aria-level="3" class="p-accordion__heading">
+                <button type="button" class="p-accordion__tab" id="header-{{id}}" aria-controls="section-{{id}}"
+                    aria-expanded="false">{{displayName}}</button>
+            </h4>
+            <section class="p-accordion__panel" id="section-{{id}}" aria-hidden="true" aria-labelledby="header-{{id}}">
+                {{#each downloads}}
+                <hr class="is-muted mb-1" />
+                <div class="flex mb-1">
+                    <div class="flex-grow">
+                        <code>{{name}}</code>
+                    </div>
+                    <div class="flex-shrink">
+                        <a class="color-link" href="{{link}}">{{../../common.download}}</a>
+                    </div>
+                </div>
+                {{/each}}
+            </section>
+        </li>
+        {{/if}}
+        {{/each}}
```

Comment:
is it possible to add changelog in this (and cli) releases page?

Reply:
I'll try.
Plus, we can always add a `View on GitHub` button.

Reply:
I think having both is the best